contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -772,6 +779,7 @@ class tsqlCommonMutator : public TSqlParserBaseListener
 public:
 	explicit tsqlCommonMutator() = default;
 	bool in_create_or_alter_function = false;
+	//bool in_execute_body_batch = false;	
```

Comment:
Let's remove it instead of commenting out.

Reply:
done